scripts/build.sh

```diff
@@ -62,8 +64,9 @@ ubuntu_setup () {
     echo "Installing Python dependencies" &&
     pip3 install --user meson ast2json &&
     meson --version &&
+    # Must disable goto contractor to enable enable old front end.
```

Comment:
Please add a reference to Github's #1110 issue to this comment, noone in the future is going to check past PRs.

Reply:
I added #1110 and a link to the issue.